### PR TITLE
Add registerAppFeature action to SDKInfoPlugin

### PR DESCRIPTION
**Pull request by ivanbogdanov**

### CODE REVIEW

File: libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/plugin/SDKInfoPlugin.java [hunk 1]
```diff
@@ -98,6 +100,24 @@ protected void getInfo(JSONArray args, final CallbackContext callbackContext) th
         }
     }
 
+    /**
+     * Native implementation for "registerSmartSyncJS" action.
```
Comment (wmathurin) [inline libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/plugin/SDKInfoPlugin.java]: Comment should say registerAppFeature

Comment (ivanbogdanov) [inline libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/plugin/SDKInfoPlugin.java]: Fixed

**Comment by ivanbogdanov** [comment]
@bhariharan

File: libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/plugin/SDKInfoPlugin.java [hunk 2]
```diff
@@ -98,6 +100,24 @@ protected void getInfo(JSONArray args, final CallbackContext callbackContext) th
...
+     * @param callbackContext Used when calling back into Javascript.
+     * @throws JSONException
+     */
+    protected void registerAppFeature(JSONArray args, final CallbackContext callbackContext) throws JSONException {
+        Log.i("SDKInfoPlugin.registerAppFeature", "registerAppFeature called");
+        // Parse args.
+        JSONObject arg0 = args.getJSONObject(0);
+        if(arg0 != null){
+            String appFeatureCode = arg0.getString("feature");
+            if(appFeatureCode != null && !appFeatureCode.isEmpty()){
```
Comment (bhariharan) [inline libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/plugin/SDKInfoPlugin.java]: You can replace this if condition with `!TextUtils.isEmpty(appFeatureCode)`.

**Comment by ivanbogdanov** [comment]
Fixed

**Pull request merged**